src/routes/(app)/admin[[hash=admin_hash]]/order/[id]/payment/[paymentId]/+page.server.ts

```diff
@@ -136,5 +138,66 @@ export const actions = {
 			});
 		}
 		throw redirect(303, request.headers.get('referer') || `${adminPrefix()}/order`);
+	},
+	replacePaymentMethod: async ({ params, request, locals }) => {
```

Comment:
The function should only be called `replace` , because there is already `payment/[paymentId]` in the url